arch/eventloop_posix_tcp.c

```diff
@@ -266,7 +268,7 @@ TCP_registerListenSocket(UA_ConnectionManager *cm, struct addrinfo *ai) {
         UA_EventLoop_registerFD(cm->eventSource.eventLoop, listenSocket,
                                 UA_POSIX_EVENT_READ,
                                 (void (*)(struct UA_EventSource *, int, void *, short))
-                                TCP_listenSocketCallback, &cm->eventSource, NULL);
+                                TCP_listenSocketCallback, &cm->eventSource, cm->initialConnectionContext);
```

Comment:
So revering to my earlier comment about the listen-socket never being called with its context: Is that why here we actually register `NULL` to be the context of the listen-socket, because it is never supposed to be called with any context.

So I can change it to `NULL` again as you had it before?

Reply:
Yeah. I think we don't need the initialContext here.
Keeping the pointer to cm/eventSource is enough.

Can be reverted to NULL.